gallery/src/data-visualization/ReactVisBarGraphPage.tsx

```diff
@@ -0,0 +1,141 @@
+        The following documentation only contains basics on how to get started with creating basic bar graphs. <br />
+        For the full, official API documentation, please click {' '}
+        <a href="https://uber.github.io/react-vis/documentation/series-reference/bar-series" target="_blank">here.</a>
+      </p>
+
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__header">
+          <h3 className="nx-h3">Data Format Reference</h3>
+        </header>
+
+        <p className="nx-p">
+          Data points for vertical bar graphs utilize the <code className="nx-code">VerticalBarSeriesPoint</code>
+          {' '} interface.
+          Data points for horizontal bar graphs utilize the <code className="nx-code">HorizontalBarSeriesPoint</code>
+          {' '} interface.
+        </p>
+
+        <NxTable>
+          <NxTableHead>
+            <NxTableRow>
+              <NxTableCell>Name</NxTableCell>
+              <NxTableCell>Type</NxTableCell>
+              <NxTableCell>Required</NxTableCell>
+              <NxTableCell>Description</NxTableCell>
+            </NxTableRow>
+          </NxTableHead>
+          <NxTableBody>
+            <NxTableRow>
+              <NxTableCell>x</NxTableCell>
+              <NxTableCell>VerticalBarSeries: string/number <br /> HorizontalBarSeries: number</NxTableCell>
+              <NxTableCell>yes</NxTableCell>
+              <NxTableCell>
+                The x position in coordinates of the box to be used.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>y</NxTableCell>
+              <NxTableCell>VerticalBarSeries: number <br /> HorizontalBarSeries: string/number</NxTableCell>
+              <NxTableCell>yes</NxTableCell>
+              <NxTableCell>
+                The y position in coordinates of the box to be used.
+              </NxTableCell>
+            </NxTableRow>
+          </NxTableBody>
+        </NxTable>
+      </section>
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__header">
+          <h3 className="nx-h3">Basic API Reference</h3>
+        </header>
+        <NxTable>
+          <NxTableHead>
+            <NxTableRow>
+              <NxTableCell>Name</NxTableCell>
+              <NxTableCell>Type</NxTableCell>
+              <NxTableCell>Required</NxTableCell>
+              <NxTableCell>Description</NxTableCell>
+            </NxTableRow>
+          </NxTableHead>
+          <NxTableBody>
+            <NxTableRow>
+              <NxTableCell>barWidth</NxTableCell>
+              <NxTableCell>number/string/object</NxTableCell>
+              <NxTableCell>No</NxTableCell>
+              <NxTableCell>
+                The percentage for which each bar fills the designated bucket. 1.0 means that the bar fills the whole
+                bucket (no padding between bars), while a smaller percentage means more whitespace between the bars.
```

Comment:
Needs to explain the format for passing an `object` here.

Reply:
> Needs to explain the format for passing an `object` here.

That was actually a typo. Should only be passing number for `barWidth` 
https://github.com/sonatype/sonatype-react-shared-components/pull/442/files#diff-01f9e2b8feee08c87c5551bc7fcfd37908269fed4267f1a52de73e7c54d8ceb0R84-R88